rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "1.62.0"
```

Comment:
We should require 1.63 which is the latest stable

Reply:
It's fine with me, since it just came out I didn't want to assume everyone is ready to update.